pom.xml

```diff
@@ -101,6 +101,15 @@
         <scope>provided</scope>
       </dependency>
       <!-- End OpenMRS core -->
+
+      <dependency>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>legacyui-omod</artifactId>
+        <version>1.0</version>
+        <type>jar</type>
+        <scope>provided</scope>
+      </dependency>
```

Comment:
Neither `<type>jar</type>` nor `<scope>provided</scope>` are really needed here. Can you check at runtime?

Reply:
@mks-d done.